frontends/web/src/contexts/DarkmodeProvider.tsx

```diff
@@ -26,7 +27,8 @@ type TProps = {
 
 export const DarkModeProvider = ({ children }: TProps) => {
   const [isDarkMode, setIsDarkMode] = useState(false);
-  const osPrefersDarkmode = useMediaQuery('(prefers-color-scheme: dark)');
+  const [osPrefersDarkMode, setOsPrefersDarkMode] = useState(false);
```

Comment:
I know it's still on draft / probably WIP, but just reminding that I don't think we need `osPrefersDarkMode` here. Can be safely removed 🙂

Reply:
You mean that we could detect the theme again in `toggleDarkmode`?

Reply:
Sorry, i wasn't being so clear. What I meant is I guess we could remove `osPrefersDarkmode` and use the `isDarkMode` instead?

The useEffect hook that sets the isDarkMode state already checks  the user's system theme and sets the state accordingly. 

```js
          detectDarkTheme().then(isDark => {
            setOsPrefersDarkMode(isDark); 
            setIsDarkMode(isDark); // here the theme on the app startup/load already determined and set to isDarkMode
          });
```

E.g

```diff
 if (runningInAndroid()) {
          preferredDarkMode = androidPrefersDarkMode;
        } else {
-         preferredDarkMode = osPrefersDarkMode;
+         preferredDarkMode = isDarkMode;
        }
```